src/biogeophys/SoilTemperatureMod.F90

```diff
@@ -1211,6 +1269,13 @@ subroutine Phasechange_beta (bounds, num_nolakec, filter_nolakec, dhsdT, &
                   t_soisno(c,j) = tfrz
                endif
 
+               ! melt excess ice after normal ice
+               if (excess_ice(c,j) > 0._r8 .AND. t_soisno(c,j) > tfrz) then
+                imelt(c,j) = 1
```

Comment:
Indent at least 2 spaces to make it more clear. We are inconsistent with our indentation, but 2 or 3 spaces is typical, and sufficient to make the indentation noticeable.

Reply:
Fixed